src/woo_publications/publications/api/filters.py

```diff
@@ -186,6 +186,18 @@ class PublicationFilterSet(FilterSet):
         widget=CSVWidget(),
         method=_filter_informatie_categorieen,
     )
+    onderwerpen = filters.ModelMultipleChoiceFilter(
+        help_text=_(
+            "Filter publications that belong to a particular topic. "
+            "When you specify multiple topics, publications belonging to any "
+            "topic are returned.\n\n"
+            "Filter values should be the UUID of the topics."
+        ),
+        field_name="onderwerpen__uuid",
```

Comment:
let's stay consistent with `informatie_categorieen` and make this:

```suggestion
        field_name="onderwerpen",
```

Reply:
![image](https://github.com/user-attachments/assets/05313782-3a5a-4bab-be3d-3543a7f47319)

With this change the filter doesn't work. Is there something I am missing?

Reply:
I'm an idiot - the python attribute controls the parameter name, not `field_name`. :)